htdocs/api/v0.0.2/candidates/visits/images/qc/QC.php

```diff
@@ -64,12 +64,8 @@ public function handleGET()
         $factory    = \NDB_Factory::singleton();
         $DB         = $factory->Database();
         $QCStatus   = $DB->pselectRow(
-            "SELECT QCStatus, 
-                pf.Value as Selected FROM files f
+            "SELECT QCStatus, Selected FROM files f
```

Comment:
Since only attributes from files_qcstatus are selected, suggest to use an imbricated query for effficiency.
```suggestion
            "SELECT QCStatus, Selected
             FROM files_qcstatus
             WHERE FileID in (
                 SELECT FileID
                 FROM files
                 WHERE File LIKE CONCAT('%', :FName)
             )",
```

Reply:
done